service-block/src/main/java/blocks/service/ServiceProperties.java

```diff
@@ -0,0 +1,71 @@
+package blocks.service;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.IntNode;
+import com.fasterxml.jackson.databind.node.LongNode;
+import com.fasterxml.jackson.databind.node.TextNode;
+
+import java.time.LocalDateTime;
+import java.time.ZonedDateTime;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+public class ServiceProperties {
+
+    public final Map<String, JsonNode> configuredProperties;
+
+    private ServiceProperties(final Map<String, JsonNode> systemProperties) {
+        this.configuredProperties = Collections.unmodifiableMap(systemProperties);
+    }
+
+    public static class Builder {
+
+        private static final Map<Class<?>, Function<Object, JsonNode>> objectToNode = new HashMap<>() {{
+            put(java.lang.String.class, o -> TextNode.valueOf((String) o));
```

Comment:
So what about LocalDate? ;) And what other types should be supported out of the box? Maybe would be useful to add own converters into the builder?